lib/facter/certbot_version.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Facter.add(:certbot_version) do
+  confine kernel: %w[FreeBSD Linux OpenBSD]
+
+  setcode do
+    value = nil
+    certbot = Facter::Util::Resolution.which('certbot')
+    if certbot
+      output = Facter::Util::Resolution.exec("#{certbot} --version 2>/dev/null")
```

Comment:
`exec` is deprecated and `execute` should now be used: https://github.com/puppetlabs/facter/blob/b0d04b9aae82c1b7f58c92326667b4e2488eaf35/lib/facter/custom_facts/core/execution.rb#L90-L101

Reply:
ah thanks, I always mix those two up.